lib/Doctrine/ORM/UnitOfWork.php

```diff
@@ -2369,6 +2369,10 @@ public function getCommitOrderCalculator()
      */
     public function clear($entityName = null)
     {
+        if ($entityName !== null && !is_string($entityName)) {
```

Comment:
I did it but then checked the file and see that it doesn't use yoda condition. Like on line below `if ($entityName === null) {` that's why I changed it :confused: